src/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,23 @@
+import './sidebar.scss';
+import React from 'react';
+interface SidebarProps {
+  onFlashcardDecksClick: (event: React.MouseEvent) => void;
+}
+export const Sidebar = ({ onFlashcardDecksClick }: SidebarProps) => {
+  return (
+    <div className="c-sidebar">
+      <label className="c-title">
```

Comment:
Let's make this `c-sidebar-title` to avoid future potential conflicts.

Reply:
good catch. fixed